JS/Game.js

```diff
@@ -1,14 +1,39 @@
 class Game {
   constructor() {
     this.round = 0,
-    this.players = {}, // {player1: 'whatever is put in imput', ...}
+    this.players = [], // {player1: 'whatever is put in imput', ...}
     this.round3 = false,
     this.winner = null
+    this.data;
```

Comment:
Is there no default value for the data on your Game class? What is this ultimately going to represent?

Reply:
Game.data becomes equal to our newly structured dataset